generator/mavgen_c.py

```diff
@@ -16,8 +20,8 @@ def generate_version_h(directory, xml):
     f = open(os.path.join(directory, "version.h"), mode='w')
     t.write(f,'''
 /** @file
- *	@brief MAVLink comm protocol built from ${basename}.xml
- *	@see http://mavlink.org
+ *  @brief MAVLink comm protocol built from ${basename}.xml
+ *  @see http://mavlink.org
  */
```

Comment:
Those are tabs, and tabs are evil ! hum by relooking at the file now, it seems that I make a mess of the indentation. Will rework that. Tabs are evil because they mix indentation ( I try to do the same on mavproxy, and break everything since tab and space are mixed ...so replace tabs by space break the syntaxe, it a nice time bomb)